security/version.go

```diff
@@ -20,6 +21,11 @@ func (v *Version) UnmarshalJSON(b []byte) error {
 	if err := json.Unmarshal(b, &tmp); err != nil {
 		return err
 	}
+
+	// be more lenient with pre-release versions, convert "2.0.0-alpha12" to "2.0.0-alpha.12"
+	re := regexp.MustCompile(`(alpha|beta)(\d+)$`)
```

Comment:
what about `rc` ?

Reply:
You're right. If a understand the [semver-specs](https://semver.org/), the pre-release-identifier could be made up of arbitrary alphanumerics, i.e. something like "1.0.0-**xyz**.9" would be valid. 

But I think that's a bit risky. We simply want to work around other packages that don't do the dot separator properly, not re-invent a complete semver-library ;-)

@fabpot should I make a new PR to add `rc` to the regexp?

Reply:
but composer itself supports `alpha`, `beta` and `rc` explicitly (and case-insensitively, which is also missing here AFAICT)

Reply:
Okay perfect, thanks. Then I'll do the same here :-)

Reply:
See #28.